features/private_datasets.feature

```diff
@@ -93,29 +93,38 @@ Feature: Private Dataset API
                 "subtopics": ["subtopic-ID"]
             }
         """
-
-    Scenario: Removing a survey from a dataset
+    
```

Comment:
Good spot! Removing dataset fields is not a current requirement for the API, because it's complicated given the way the PUT endpoint is written. But we are planning a general refactor of put/patch functionality after the go live next week.

Reply:
Thanks for explaining, that's interesting to know.